### DISCUSSION TITLE
Similar boilerplate in typescript

### DISCUSSION THREAD

**Discussion by samkit5495** [hagopj13/node-express-boilerplate#95] [Q&A]
Do we have similar boilerplate in typescript ?

**Comment by trasherdk** [comment]
Here's 2 I was thinking about checking out.

https://github.com/excelbrium/ts-express-boilerplate
https://github.com/excelbrium/ts-boilerplate

**Reply by markvital** [reply]
Someone ask similar question here: https://github.com/hagopj13/node-express-boilerplate/issues/136. In the issue there is a link to fork with Typescript:
https://github.com/dominick-more/node-express-typescript-boilerplate/tree/pr/typescript-conversion

Also there is PR: https://github.com/hagopj13/node-express-boilerplate/pull/153, but it seems unfinished
